modules/kernel/src/org/apache/axis2/classloader/BeanInfoCache.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@kasunbg : The reason for having this license is because these classes are from the fix from the Axis2 repo.